Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gemspec
+gem 'shanty', path: '/users/Chris/shanty/shanty'
```

Comment:
Should remove the path from here, and instead use the Bundler config trick to make it come from local.

```ruby
gem 'shanty', git: 'https://github.com/shanty/shantytown'
```

```sh
bundle config local.shanty ~/shanty/shanty
```